ts2/src/components/Chart.tsx

```diff
@@ -0,0 +1,107 @@
+import * as d3 from 'd3'
+import * as React from 'react';
+import * as T from './Chart.types'
+import styles from './Chart.module.css'
+
+export const Component: React.FunctionComponent<T.IProps> = (props) => {
+
+    const file = props.filename
+    const country = props.country
+    const svgHeight = props.height
+    const svgWidth = props.width
```

Comment:
Предложу обратить внимание не деструктуризацию:
```js
    const file = props.filename
    const country = props.country
    const svgHeight = props.height
    const svgWidth = props.width
```
то же самое:
```suggestion
export const Component: React.FunctionComponent<T.IProps> = ({
  country,
  filename: file,
  height: svgHeight,
  width: svgWidth,
}) => {
```
[Подробный гайд по деструктуризации](https://exploringjs.com/es6/ch_destructuring.html)

Reply:
Хорошо, спасибо!